app/assets/api/posts.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": 1,
+    "title": "Routing using elm-route",
+    "body": "Read more about `Bogdanp/elm-route` at http://package.elm-lang.org/packages/Bogdanp/elm-route/latest"
```

Comment:
It is actually trivial doing so with `elm-markdown`. Even though the example is growing by the day, it seems like a worthy addition.